net.pioneerspacesim.Pioneer.json

```diff
@@ -0,0 +1,102 @@
+{
+	"app-id": "net.pioneerspacesim.Pioneer",
+	"runtime": "org.freedesktop.Platform",
+	"runtime-version": "1.6",
+	"sdk": "org.freedesktop.Sdk",
+	"command": "pioneer",
+
+	"finish-args": [
+		"--share=ipc",
+		"--socket=x11",
+		"--socket=wayland",
+		"--socket=pulseaudio",
+		"--device=dri",
+		"--persist=.pioneer"
```

Comment:
Should be fixed upstream (and for flathub purposes, at least reported) to use $XDG_CONFIG_DIR.

Reply:
Well I don't have much control over that... And I doubt upstream will accept to change the savestates directory, that would confuse a lot of people.

Reply:
It's not really rocket science though to read old saves from old directory, but use a correct one when there is no old save games. Please report it regardless.

Reply:
Reported here: https://github.com/pioneerspacesim/pioneer/issues/4392